tests/test_AggregateMSWData.cpp

```diff
@@ -142,29 +152,117 @@ Opm::data::Wells wr()
     }
     return xw;
 }
+
+//------------------------------------------------------------------+
+// Models a multi-lateral well with the following segment structure |
+//------------------------------------------------------------------+
+//                                                                  |
+//                     12   13    14     15     16                  |
+//                   o----o----o-----o------o-----o (2)             |
+//              11  /              20 \   21 \                      |
+//                 /                   o      o (6)                 |
+//                /                     \                           |
+//               /                    22 \  23   24                 |
+//   1   2   3  /  4   5   6              o----o----o (5)           |
+//  ---o---o---o-----o---o---o (1)                                  |
+//                        \                                         |
+//                       7 \  8   9    10                           |
+//                          o---o---o-----o (3)                     |
+//                                         \                        |
+//                                       17 \   18   19             |
+//                                           o----o----o (4)        |
+//------------------------------------------------------------------+
+//  Branch (1):  1,  2,  3,  4,  5,  6                              |
+//  Branch (2): 11, 12, 13, 14, 15, 16                              |
+//  Branch (3):  7,  8,  9, 10                                      |
+//  Branch (4): 17, 18, 19                                          |
+//  Branch (5): 20, 22, 23, 24                                      |
+//  Branch (6): 21                                                  |
+//------------------------------------------------------------------+
```

Comment:
Nice documentation,

Reply:
> Nice documentation,

Thanks.  I didn't particularly like the right-hand border, but it was the easiest way of suppressing diagnostic messages about `\` generating multi-line comments...